ext/mbstring/php_unicode.c

```diff
@@ -238,6 +238,41 @@ static uint32_t *emit_special_casing_sequence(uint32_t w, uint32_t *out)
 	return out;
 }
 
+/* Used when determining whether special casing rules should be applied to Greek letter sigma */
+static bool scan_ahead_for_cased_letter(unsigned char *in, size_t in_len, unsigned int state, const mbfl_encoding *encoding)
+{
+	uint32_t wchar_buf[64];
+
+	while (in_len) {
+		size_t out_len = encoding->to_wchar(&in, &in_len, wchar_buf, 64, &state);
+		ZEND_ASSERT(out_len <= 64);
+		for (int i = 0; i < out_len; i++) {
```

Comment:
Nit: implicit unsigned to signed conversion
```suggestion
		for (unsigned int i = 0; i < out_len; i++) {
```

Reply:
Thanks, fixed.